src/utilities/SSRB.cxx

```diff
@@ -96,19 +91,56 @@ int main(int argc, char **argv)
 	   << "max_in_segment_num_to_process defaults to all segments\n";
       exit(EXIT_FAILURE);
     }
-  const string  output_filename = argv[1];
+
+  // Define defaults
+  int span = 1;
+  int num_views_to_combine = 1;
+  int max_segment_num_to_process = -1;
+  bool do_norm = true;
+  int num_tangential_poss_to_trim = 0;
+
+  const string output_filename = argv[1];
   shared_ptr<ProjData> in_projdata_ptr = ProjData::read_from_file(argv[2]);
-  const int span = argc<=3?1 : atoi(argv[3]);
-  const int num_views_to_combine = argc<=4 ? 1 : atoi(argv[4]);
-  const bool do_norm = argc<=5 ? true : atoi(argv[5]) != 0;
-  const int max_segment_num_to_process = argc <=6 ? -1 : atoi(argv[6]);
-  SSRB(output_filename,
-       *in_projdata_ptr,
-       span,
-       num_views_to_combine,
-       num_tangential_poss_to_trim,
-       do_norm,
-       max_segment_num_to_process
-       );
+
+  if (strcmp(argv[3], "--template")==0)
```

Comment:
please follow same conventions as `zoom_image`, i.e. all options upfront, and hence `SSRB --template bla output input`.

Also, will need to check if `argc` is appropriate for this signature.